iceoryx_posh/test/integrationtests/test_client_server.cpp

```diff
@@ -271,7 +271,8 @@ TEST_F(ClientServer_test, SlowServerDoesNotBlockClient)
     Server<DummyRequest, DummyResponse> server{sd, serverOptions};
 
     // send request till queue overflows
-    for (uint64_t i = 0; i < serverOptions.requestQueueCapacity + NUMBER_OF_OVERFLOWS; ++i)
+    const int64_t iMax = static_cast<int64_t>(serverOptions.requestQueueCapacity) + NUMBER_OF_OVERFLOWS;
```

Comment:
I think we could avoid the problem entirely if `NUMBER_OF_OVERFLOWS` would be `uint64_t` instead?

Logically the tests should be still ok with this change.

Reply:
... oh well